src/main/java/com/lambdatest/jenkins/freestyle/MagicPlugDescriptor.java

```diff
@@ -27,6 +30,8 @@
 @Extension
 public class MagicPlugDescriptor extends BuildWrapperDescriptor {
 
+	private final static Logger logger = Logger.getLogger(AppAutomationCapabilityService.class.getName());
```

Comment:
please change the class name in logger instance to, `MagicPlugDescriptor`

Reply:
Done